python/cugraph/cugraph/link_analysis/pagerank.py

```diff
@@ -89,24 +103,41 @@ def pagerank(
     PageRank : cudf.DataFrame
         GPU data frame containing two cudf.Series of size V: the vertex
         identifiers and the corresponding PageRank values.
-
         df['vertex'] : cudf.Series
             Contains the vertex identifiers
         df['pagerank'] : cudf.Series
             Contains the PageRank score
```

Comment:
We should probably also include the following as part of the description of the return value:

`NOTE: if the input cugraph.Graph was created using the renumber=False option of any of the from_*_edgelist() methods, pagerank assumes that the vertices in the edgelist are contiguous and start from 0. If the actual set of vertices in the edgelist is not contiguous (has gaps) or does not start from zero, pagerank will assume the "missing" vertices are isolated vertices in the graph, and will compute and return pagerank values for each.`